frontend/lib/DataModels/User/user_role.dart

```diff
@@ -0,0 +1,28 @@
+enum UserRole {
+  user,
+  admin,
+  empty,
+}
+
+extension UserRoleExtension on UserRole {
+  String get roleName {
+    switch (this) {
+      case UserRole.user:
+        return "USER";
+      case UserRole.admin:
+        return "ADMIN";
+      case UserRole.empty:
+        return "EMPTY";
+    }
+  }
+
+  static UserRole userRole(String roleString) {
```

Comment:
Czy ta funkcja nie powinna nazywać się `fromString`?

Reply:
imo mogłoby być cos typu 
extension StringToRole on String {
  UserRole get role {
    switch (this) {
      case 'USER':
        return UserRole.user;
      case 'ADMIN':
        return UserRole.admin;
    }
    return UserRole.none;
  }
}

Reply:
jezus ale brzydko sformatowało

Reply:
Teraz będzie ładniej XD